src/components/FilterInput/index.tsx

```diff
@@ -1,45 +1,48 @@
-import React, { ReactElement, useEffect, useState } from 'react';
+import React, { ReactElement, useState } from 'react';
 import { SafeAreaView } from 'react-native';
 
 import { Picker } from '@react-native-community/picker';
-
 import { ItemValue } from '@react-native-community/picker/typings/Picker';
-import { ContainerPicker, Title, PickerStyle } from './styles';
-
-interface FilterInputProps {
-  title: string;
-  items: Array<string>;
-}
+import { ContainerPicker, PickerStyle } from './styles';
 
-interface IStateProps {
-  label: string;
+interface IFilterProps {
   id: number;
+  name: string;
 }
 
-export function FilterInput({ title, items }: FilterInputProps): ReactElement {
-  const [stateIndexSelected, setStateIndexSelected] = useState<ItemValue>();
+export function FilterInput(): ReactElement {
+  const [typeSelected, setTypeSelected] = useState<ItemValue>();
 
-  const [label, setLabel] = useState(items);
+  const constructionTypes: IFilterProps[] = [
```

Comment:
The ideal would be to pass the data as a component attribute, so we make this component much more reusable and dynamic